{% extends "layout.html" %}
  
{% block title %}{% endblock %}
{% block lead %}

<div class="d-flex justify-content-center">
  <div class="col igemthrace-image-container" style="background-image: url('https://static.igem.wiki/teams/4989/wiki/wiki/gear.gif');background-repeat: no-repeat;
  background-size: cover;
  background-position: center;
  background-attachment: fixed;">
    <div class="igemthrace-blur d-flex justify-content-center" style="width: 100%;height: 100%;align-items: center;">
      <h2 class="igemthrace-white-text-coloring igemthrace-bold-font">See our engineering success !</h2>
    </div>
  </div>
</div>

<!--
Demonstrate engineering success in a part of your project by going through at least one iteration of the engineering design cycle.
-->
{% endblock %}

{% block page_content %}

<div class="container">
  <div class="row igemthrace-blue-shadow">
    <h2 class="igemthrace-blue-text-coloring">Introduction</h2>
    <p class="igemthrace-blue-text-coloring">Engineering success lies in the main principle of starting to build a synthetic biological system, a dry lab model or other analysis, a hardware or software project. It discloses the fundamental requirements that should be met in order for the engineering process to be more valuable, effectively implemented and foster interdisciplinary collaboration.</p>
    <p class="igemthrace-blue-text-coloring">The Design, Built, Test, Learn cycle (DBTL) is a fundamental concept in engineering, particularly in the field of synthetic biology and bioengineering. This iterative process guides engineers and scientists in developing novel biological systems. </p>
    
    <div style="text-align: center;">
      <figure>
        <img src="https://static.igem.wiki/teams/4989/wiki/athanasia-dbtl-2.png" style="width: 600px;">
        <figcaption></figcaption>
      </figure>
    </div>
  
    <p class="igemthrace-blue-text-coloring">Ιn the "Design" phase, researchers conceptualize and plan their engineered biological systems, specifying the genetic components and functions required. The "Build" phase involves physically constructing the designed genetic constructs in the laboratory. In the "Test" phase, the functionality and performance of the engineered system are assessed, often through experimental testing. Finally, the "Learn" phase interprets the results, identifies areas for improvement, and refines the design for the next cycle.</p>
    <p class="igemthrace-blue-text-coloring">The DBTL cycle is a powerful approach that not only accelerates the development of new biological solutions but also facilitates continuous learning and optimization, making it an indispensable tool in the field of bioengineering.</p>
    <p class="igemthrace-blue-text-coloring">As a team from our very early steps in Igem’s synthetic biology project, we aimed to combine the engineering circle and integrated human practices in order to configure our project. Engineering success was accomplished in different aspects of our project such as the wet lab, dry lab, and our software tool. In this way, we made 3 interactive engineering circles that the progress of the one influences and potentially changes the progress of the other.</p>
  </div>
  <div class="row d-flex justify-content-center">
    <div class="igemthrace-engineering-button" onclick="show_wet_lab_aspect()">
      <h3>WET LAB ASPECT</h3>
    </div>
    <div class="igemthrace-engineering-button" onclick="show_dry_lab_aspect()">
      <h3>DRY LAB ASPECT</h3>
    </div>
    <div class="igemthrace-engineering-button" onclick="show_software_aspect()">
      <h3>SOFTWARE ASPECT</h3>
    </div>
  </div>

  

  <div id="wet-lab-aspect" class="row igemthrace-is-hidden igemthrace-blue-shadow">
    <h3 class="igemthrace-blue-text-coloring" style="text-align: center;">WET LAB</h3>
    <!-- Wet Lab Text -->
    <p class="igemthrace-blue-text-coloring">For implementing the DBTL cycle on our wet lab projects we separated our tasks into two different categories, always based on the gold medal criterion of creating a new improved part:</p>

    <ul><ul>
      <li class="igemthrace-blue-text-coloring">TESTING THE OLD PART BBa_K1618021</li>
      <li class="igemthrace-blue-text-coloring">TESTING THE NEW PART BBa_K4989008</li>
    </ul></ul>
  
    <p></p>    
    <h3><strong>1ST CYCLE----Testing the old part</strong></h3>
    <p></p>        
    <h3><strong>DESIGN</strong></h3>
    <p class="igemthrace-blue-text-coloring">After we concluded with the conduction of bibliographical research about the butyrate-producing pathway we searched in the registry part about a part that can produce butyrate. Simultaneously, we designed our target ideal gene cluster and we compared it with the part that we found in the registry (BBA_K1618021). We realized that it needed certain changes in order to comply with our needs for producing butyrate. Thus, we proceeded with an in-silico analysis of the sequence. Since it is a testing for the function of this sequence we did not make any changes.</p>
    <p class="igemthrace-blue-text-coloring">Since we wanted the sequence to be synthesized and cloned into a vector, we deposited the sequence’s part to IDT as it is in order for it to be cloned into a GoldenGate vector. Then we would transform with this vector 3 Lactobacillus species and E.coli K-12. We didn’t design a different vector since we only aimed to test the part.</p>
    <p class="igemthrace-blue-text-coloring">Of course, we had designed our experiment guidelines and steps and collected all of our reagents. Also, we recorded the results that we expected to see and the reasons behind them.</p>
    <p></p>        
    <h3 style="color: red;">CROSSROAD</h3>
    <p class="igemthrace-blue-text-coloring">This type of sequence was too big in size in order for it to be synthesized. Thus we needed to re-design it in gBlocks and add restriction sites on each of its edges to perform later a Golden Gate assembly.</p>
    <p></p>    
    <h3>1st RE-DESIGN</h3>
    <p class="igemthrace-blue-text-coloring">We performed an in-sillico analysis again and we “optimized” our sequence in order to not include any unwanted restriction sites of the endonucleases that we wanted to use. Also, in our sequences, we designed and added the target restriction sites in each of the two edges of the gBlocks. Then we performed an in-sillico-based digestion in order to see that everything was designed as wanted. Also, we needed to find a new vector for cloning.</p>
    <p></p>    
    <h3 style="color: red;">CROSSROAD</h3>   
    <p class="igemthrace-blue-text-coloring">Since the vector Pmg36e was not available for utilization we decided to proceed with the pNZ8149 plasmid. This vector also gave a solution for cloning and testing the old part.</p>
    <p></p>    
    <h3><strong>BUILD</strong></h3>
    <p class="igemthrace-blue-text-coloring">After IDT synthesized our gBlocks, we performed GoldenGate assembly in order to assemble the parts together. Then we proceeded to ligate this part with the pNZ vector and with this we transformed the competent cells of Lactobacillus and E.coli strains. This is how we built our microbial chassis in order for them to potentially express butyrate.</p>
    <p></p>    
    <h3><strong>TEST</strong></h3>
    <p class="igemthrace-blue-text-coloring">We chose those clones that included the insert part, and cultured them in order to give them time to produce the corresponding portions of butyrate. When we measured, through HPLC, the concentrations of butyrate, we found out that there wasn’t any level of butyrate production in Lactobacillus and the average butyrate production in E.coli. </p>
    <p></p>    
    <h3><strong>LEARN</strong></h3>
    <p class="igemthrace-blue-text-coloring">We analyzed those results and we discussed what we have learned if they verified our recorded predictions before conducting the experiment, and what we should do in order to produce butyrate.</p>
    <p class="igemthrace-blue-text-coloring">Our 1st DBTL circle was performed and concluded successfully. We received very promising results since the first part wasn’t functional and we got the chance to re-design it based on the notes we made on why we expected the experiment to not work. Therefore we were reassured of the results that we awaited and we confirmed experimentally that our target host Lactobacillus rhamnosus GG does not produce butyrate with the old part.</p>


    <p></p>    
    <h3><strong>2ND CYCLE----TESTING THE NEW PART</strong></h3>
    <p></p>    
    <h3 style="color: red;">CIRCLE INTERSECTION</h3>
    <p class="igemthrace-blue-text-coloring">Based on the results from the second circle, we began the design of the new improved part.</p>
    <p></p>    
    <h3><strong>DESIGN</strong></h3>
    <p class="igemthrace-blue-text-coloring">We wanted to design our new part in order to have some basic characteristics that resembled an old part just so we keep the general structure. We applied all of the changes in the genetic makeup and the regulatory characteristics and we ran the sequence through in-sillico analyses. Our goal was to synthesize this part inside the Pmg36e plasmid. So we included the Biobrick scars amongst the genes and the RBSs as Igem requires. Also, we had all the enzymes, the reagents, and the protocols ready for use.</p>
    <p></p>    
    <h3 style="color: red;">CROSSROAD</h3>
    <p class="igemthrace-blue-text-coloring">Since the vector Pmg36e  was not available for utilization we decided to synthesize our sequence from IDT in a GoldenGate plasmid.</p>
    <p class="igemthrace-blue-text-coloring">However the delay of the part delivery was prohibitive to order it, so we decided to re-design the sequence in different gBlocks.</p>

    <p></p>        
    <h3><strong>1st RE-DESIGN</strong></h3>
    <p></p>    
    <p class="igemthrace-blue-text-coloring">We redesigned our part in sillico and we separated the parts safely with the appropriate type IIS restriction sites on the edges, for GoldenGate assembly. However, there were some technical issues with the sequence and due to the terminator and other parts of the sequences unfortunately we could not synthesize our part from IDT.</p>
    <p></p>    
    <h3 style="color: red;">CIRCLE INTERSECTION</h3>
    <p class="igemthrace-blue-text-coloring">The Pnz8149 vector was finally available from our lab so we had the ability to do the cloning on our own. We approached Twist and ordered our gBlock sequences which passed the technical control.</p>
    <p class="igemthrace-blue-text-coloring">Unfortunately due to the big delay with the pMG36e plasmid synthesis and the difficulty that came up with our part’s order, we could not test eventually the new improved part. This was due to the fact that the parts from the Twist would be delivered just for days after the Wiki Freeze. So besides the design level, we could not go anywhere further in the DBTL cycle.</p>
    <p class="igemthrace-blue-text-coloring">However, we represent below the rest of the DBTL cycle that could possibly take place.</p>
    <p></p>    
    <h3><strong>BUILD</strong></h3>
    <p class="igemthrace-blue-text-coloring">After the obtainment of the parts, we performed the corresponding digestions for the GoldenGate assembly of the parts. Then the whole insert is cloned into the pNZ vector and with this we transformed the Lactobacillus species and the E.coli K-12 strain in order to create one chassis for the proof of our concept and 3 other chassis for our part’s characterization.</p>
    <p></p>    
    <h3><strong>TEST</strong></h3>
    <p class="igemthrace-blue-text-coloring">Since we added the GFP molecule in our insert we could isolate the colonies with the recombinant plasmid that has our part, and culture it again in order to give time to the bacteria to grow and produce butyrate.</p>
    <p class="igemthrace-blue-text-coloring">Then, we measured with HPLC the levels of butyrate and acetate, since the latter is explicitly related to the butyrate production.</p>
    <p></p>    
    <h3><strong>LEARN</strong></h3>    
    <p class="igemthrace-blue-text-coloring">At the learning level we could have two possible scenarios:</p>
    
    <ul><ul>
      <li class="igemthrace-blue-text-coloring">If the butyrate levels are high in comparison with our controls and the acetate levels are relatively low since it has been utilized, then our new improved part is working and our DBTL cycle is successful!</li>
      <li class="igemthrace-blue-text-coloring">If we do not detect any butyrate levels or they are very low in comparison with our controls then our part did not function as we expected. Then a new DBTL cycle would start again with the design part stage and correct any possible omissions or mistakes. Besides the design of the part faults, the inefficient production of our part could be attributed to the built stage. Our strain “hosts” a new metabolic pathway that could unbalance the overall metabolism. Thus, it could be necessary for our organisms that would accept this plasmid to be genetically engineered directly to their genetic makeup in order to “redirect” some crucial pathways of the metabolism. This could be done with the deletion of a gene that encodes for an enzyme that “opens” up a new pathway from the utilization of acetyl-CoA.</li>
    </ul></ul>
    
    <p class="igemthrace-blue-text-coloring">We us a team decided to put all our efforts in, to test and characterize our new improved part, because we have a strong belief in its Design level. So we aiming to produce our results, build our chassis and evaluate our results, after the Wiki Freeze, in order to present them to in our Q&A session to the judges. Be sure that we would come up with some very exciting results!</p>
    
    <!-- end of Wet Lab Text -->
  </div>
  


  

  <div id="dry-lab-aspect" class="row igemthrace-is-hidden igemthrace-blue-shadow">
    <h3 class="igemthrace-blue-text-coloring" style="text-align: center;">DRY LAB</h3>
    <!-- Dry Lab Text -->
    <p class="igemthrace-blue-text-coloring">To excel on every project on the dry lab level, it is equally important to apply the DBTL principles. The changes that arose at each level, came from experienced scientists and analyses in the field of bioinformatics.</p>
    <p></p>    
    <h3><strong>DESIGN</strong></h3>    
    <p class="igemthrace-blue-text-coloring">After reviewing available metabolic modeling software, we decided to use MICOM. MICOM is a Python package for the metabolic modeling of microbial communities. In this way, we wanted to design an in sillico microbial community that could resemble those of the gut flora and then potentially test any changes that could bring abnormalities to the microbial synthesis.</p>
    <p></p>    
    <h3><strong>BUILD</strong></h3>    
    <p class="igemthrace-blue-text-coloring">The construct of a community model is conducted from a list of input COBRA models and manages exchange fluxes between individuals and individuals with the environment.</p>
    <p></p>    
    <h3 style="color: red;">CROSSROAD</h3>
    <p class="igemthrace-blue-text-coloring">After discussing this software with the experienced bioinformaticist Haris Zafeiropoulos, PhD, he advised us not to use MICOM, as it does not fulfill our needs of observing the flux changes of SCFA's in gut microbiome through time. He advised us to design a bioinformatics workflow, to in-silico reconstruct the L.Rhamnosus GG genome and see the change in SCFA fluxes.</p>
    <p class="igemthrace-blue-text-coloring">However, we did not have sufficient time to complete such a workflow. We decided to design another workflow, for the identification of bacteria species and enzymes differentially enriched with significant statistical differences. This way, we could propose possible enzymes affecting butyrate production to our wet lab, as candidates to be expressed in our potential vector system.</p>
    <p class="igemthrace-blue-text-coloring">As the first engineering cycle wasn’t successfully fulfilled we proceeded with a new one, which was very important for the experimental designs of the wet lab.</p>
    <p></p>    
    <h3><strong>DESIGN</strong></h3>    
    <p class="igemthrace-blue-text-coloring">At first, our wet lab team was keen on expressing butyrate kinase through a vector system, as this enzyme is crucial to butyrate production and we found absence of it in the L. Rhamnosus GG genome through genome annotation.</p>
    <p></p>    
    <h3><strong>BUILD</strong></h3>    
    <p class="igemthrace-blue-text-coloring">We built a bioinformatics workflow, from bacteria cohort creation to genome annotation. Thus, we tested if there are other possible genes, with a potential role in the butyrate pathway.</p>
    <p></p>    
    <h3><strong>TEST</strong></h3>
    <p class="igemthrace-blue-text-coloring">We found other enzymes, contributing to butyrate synthesis, way more enriched than butyrate kinase such as butyryl-CoA:acetyl-CoA transferase or butyryl-CoA Dehydrogenase.</p>
    <p></p>    
    <h3><strong>LEARN</strong></h3>    
    <p class="igemthrace-blue-text-coloring">Through bibliography, we studied the role of transferase and a possible in-vivo reconstruction on L.Rhamnosus GG with a sequential catalyzed protein reaction.</p>    
    <p class="igemthrace-blue-text-coloring">We compared the pros and cons of adding a kinase and a transferase. First, transferase is found to be more decreased in the decreased bacteria cohort from people with depression . It also utilizes the acetate that is already produced in high quantities from the Lactobacillus rhamnosus GG. This makes it the ideal pathway to add to our butyrate-producing metabolic pathway.</p>
        
    <!-- end of Dry Lab Text -->
  </div>
  

  


  <div id="software-aspect" class="row igemthrace-is-hidden igemthrace-blue-shadow">
    <h3 class="igemthrace-blue-text-coloring" style="text-align: center;">SOFTWARE</h3>
    <!-- Software Text -->
    <p class="igemthrace-blue-text-coloring">Even in the process of software development, the DBTL cycle was fundamental, and the feedback that changed the cycles multiple times in order to have our ideal result came from our integrated human practices work.</p>
    <p></p>    
    <h3><strong>DESIGN</strong></h3>    
    <p class="igemthrace-blue-text-coloring">Our initial thought was to develop an app that could combine therapeutics with diagnostics. Thus, we decided to create a chatbot that could be available for the patient to answer questions and evaluate if the answers are pruned to be categorized as depressive. Then, if this is the case the chatbot will advise the user to visit a therapist for a psychological evaluation/consultation.</p>
    <p class="igemthrace-blue-text-coloring">The tools used for this project are based on AI. More specifically its fundaments lie to neural networks that can classify the texts as depressive or non-depressive.</p>
    <p></p>    
    <h3><strong>BUILD</strong></h3>    
    <p class="igemthrace-blue-text-coloring">The whole code section was built in Flutter environment. The neural architecture was designed and trained using tensorflow.</p>
    <p></p>        
    <h3><strong>TEST</strong></h3>    
    <p class="igemthrace-blue-text-coloring">After the creation of the app we tested it and the results were very promising. We tried key phrases that we knew were classified as depressive and indeed the chatbot advised us to visit a therapist.</p>
    <p></p>        
    <h3><strong>LEARN</strong></h3>    
    <p class="igemthrace-blue-text-coloring">Since the results were those that we expected the app functioned as we planned. The only concerns were about its enrichment and the further development of the graphics.</p>
    
    <p></p>        
    <h3 style="color: darkblue;">HUMAN PRACTICES INTERSECTION</h3>
    <p></p>        
    <h3><strong>1st RE-DESIGN</strong></h3>    
    <p class="igemthrace-blue-text-coloring">After a presentation of our app to Assistant Professor of Nutrition and Dietetics Dr.Anastasiou he advised us to link the app with our project since the app provided only prompts for the patient to consult a therapist.</p>        
    <p>We included in our app the aspect of the diet. More specifically, we added the characteristic of tracking the user’s dietary habits and linked those with the concentration of the SCFAs. Simultaneously, we got valuable advice and a detailed dietary guide on food rich in SCFAs from our experienced Nutritionist, Elpida Papanikolaou.</p>            
    <h3><strong>1st RE-BUILD</strong></h3>    
    <p class="igemthrace-blue-text-coloring">We developed a local database in order to keep track of the user’s dietary habits. This was implemented with sqllite.</p>        
    <h3><strong>1st RE-TEST</strong></h3>    
    <p class="igemthrace-blue-text-coloring">We tested again our app, ensuring that both databases where functional (text and diet database).</p>        
    <h3><strong>1st RE-LEARN</strong></h3>    
    <p class="igemthrace-blue-text-coloring">Since the application was effective we didn’t have to change and re-design any part of our application. However, after a meeting with the  Group of Psychiatrists of the Society of Social Psychiatry P. Sakellaropoulos, new enhancements that should be introduced to the app came up, opening a new DBTL cycle.</p>        

  
    <p></p>        
    <h3 style="color: darkblue;">HUMAN PRACTICES INTERSECTION</h3>
    <p></p>        
    <h3><strong>2st RE-DESIGN</strong></h3>    
    <p class="igemthrace-blue-text-coloring">Based on the meeting with the  Group of Psychiatrists of the Society of Social Psychiatry P. Sakellaropoulos, we made some fundamental changes on the basis of the application. First of all we needed to change the source of the questionnaire that the chatbot uses for answering the texts given by the user. We included the QIDS-SR (Quick Inventory of Depressive Symptomatology) that is widely used in the field of depression detection and therapy.</p>        
    <p class="igemthrace-blue-text-coloring">Secondly, they recommended us to adjust the app in order to recommend nearby mental health specialists, provided that the text is classified as depressive. Also, we included the feature of the storage of previous conversation between the user and the chatbot. In this way the user will pick a date and the corresponding conversation will be shown.</p>            
    <h3><strong>2st RE-BUILD</strong></h3>    
    <p class="igemthrace-blue-text-coloring">We redesigned and implemented the UI of the application. We changed the list of strings that where stored into the database as questions and we use the google API for maps in order to recommend nearby experts</p>        
    
    <h3><strong>2st RE-TEST</strong></h3>    
    <p class="igemthrace-blue-text-coloring">After including those features we tested the app to see if the are incorporated into it. We tested the google api separately while we also printed multiple times the contents of the database to ensure that the new questions are stored.</p>        
    
    <h3><strong>2st RE-LEARN</strong></h3>    
    <p class="igemthrace-blue-text-coloring">Since the application was effective we didn’t have to change and re-design any part of our application.</p>        
    <p class="igemthrace-blue-text-coloring">Lastly, after creating a questionnaire for the public on their opinion about our application, there were many concerns about the accuracy of the chatbox on discriminating the depressive from the non-depressive texts.</p>        



    <p></p>        
    <h3 style="color: darkblue;">HUMAN PRACTICES INTERSECTION</h3>
    <p></p>        
    <h3><strong>3st RE-DESIGN</strong></h3>    
    <p class="igemthrace-blue-text-coloring">We collected a very large dataset with depressive or non-depressive texts from multiple scientific sources. In this way we tried to train our model more effectively in order for it to be more accurate.</p>        
    
    <h3><strong>3st RE-BUILD</strong></h3>    
    <p class="igemthrace-blue-text-coloring">We redesigned the dense part of our neural architecture. We increased the dropout as the change on the training data resulted in some overfitting.</p>        
    
    <h3><strong>3st RE-TEST</strong></h3>    
    <p class="igemthrace-blue-text-coloring">We checked the efficiency of our model using AUC and logloss metrics. We got better results. AUC was around 0.8 while logloss was around 0.3</p>        
    
    <h3><strong>3st RE-LEARN</strong></h3>    
    <p class="igemthrace-blue-text-coloring">Since the application was effective enough we didn’t have to change and re-design the neural architecture.</p>        

    <p class="igemthrace-blue-text-coloring">In this way, we concluded with the development of DEPRETECT and we accomplished to excel every nuance of the application.</p>        

    <!-- end of Software Text -->
  </div>
</div>





<script>
  function show_software_aspect() {
    soft =  document.getElementById('software-aspect');
    wet = document.getElementById('wet-lab-aspect');                 
    dry = document.getElementById('dry-lab-aspect'); 
    soft.classList.remove('igemthrace-is-hidden'); 
    wet.classList.add('igemthrace-is-hidden');
    dry.classList.add('igemthrace-is-hidden');                                             
  }

  function show_wet_lab_aspect() {
    soft =  document.getElementById('software-aspect');
    wet = document.getElementById('wet-lab-aspect');                 
    dry = document.getElementById('dry-lab-aspect'); 
    soft.classList.add('igemthrace-is-hidden'); 
    wet.classList.remove('igemthrace-is-hidden');
    dry.classList.add('igemthrace-is-hidden');                                             
  }

  function show_dry_lab_aspect() {
    soft =  document.getElementById('software-aspect');
    wet = document.getElementById('wet-lab-aspect');                 
    dry = document.getElementById('dry-lab-aspect'); 
    soft.classList.add('igemthrace-is-hidden'); 
    wet.classList.add('igemthrace-is-hidden');
    dry.classList.remove('igemthrace-is-hidden');                                             
  }
</script>

<!--
<div class="row mt-4">
  <div class="col">
    <div class="bd-callout bd-callout-info">
      <h4>Silver Medal Criterion #1</h4>
      <p>Demonstrate engineering success in a part of your project by going through at least one iteration of the engineering design cycle. This achievement should be distinct from your Contribution for Bronze.<p>
      <p>If you plan to show engineering success by creating a new Part that has been shown to work as expected, you must document your contribution on the Part's Main Page on the <a href="http://parts.igem.org/Main_Page">Registry</a> for your team to be eligible for this criteria.</p>
      <hr>
      <p>Please see the <a href="https://competition.igem.org/judging/medals">2023 Medals Page</a> for more information.</p>
    </div>
  </div>
</div>
-->
{% endblock %}
