{% extends "layout.html" %}
  
{% block title %}{% endblock %}
{% block lead %}

<div class="d-flex justify-content-center">
  <div class="col igemthrace-image-container" style="background-image: url('https://static.igem.wiki/teams/4989/wiki/wiki/new-improved-part.gif');background-repeat: no-repeat;
  background-size: cover;
  background-position: top;
  background-attachment: fixed;">
    <div class="igemthrace-blur d-flex justify-content-center" style="width: 100%;height: 100%;align-items: center;">
      <h2 class="igemthrace-white-text-coloring igemthrace-bold-font">New improved part - </h2>
      <h2 class="igemthrace-white-text-coloring igemthrace-bold-font">Gold Criterion</h2>
    </div>
  </div>
</div>

<!--
Demonstrate engineering success in a part of your project by going through at least one iteration of the engineering design cycle.
-->
{% endblock %}

{% block page_content %}

<div class="container d-flex flex-column">
  <div class="row igemthrace-blue-shadow p-3 rounded">
    <h2 class="igemthrace-blue-text-coloring">Introduction</h2>
    <p class="igemthrace-blue-text-coloring">Our project DEPREGUT, is based on a thorough design both of a metabolic pathway and a whole food-grade system. First, the metabolic pathway of butyrate production serves our goal of producing the therapeutic target biomolecule that will interact with the enterochromaffin cells in the gut and elevate the production of serotonin. Secondly, in order to take our project and develop it outside of the lab we need to meet some specific requirements. These apply to creating a food-grade system that will ensure the consumer’s safety and enhance the quality of our probiotic.</p>
    <p class="igemthrace-blue-text-coloring">For this year’s gold medal, besides our nominations for specializations, we chose to compete for the New Improved Part from General Biological Engineering.</p>

  </div>

  <div class="igemthrace-space-height-2"></div>

  <div class="row igemthrace-blue-shadow p-3 rounded">
      <h2 class="igemthrace-blue-text-coloring">Engineering of the metabolic pathway-coding sequences</h2>
      <p class="igemthrace-blue-text-coloring">Butyrate production is closely related to the glycolysis metabolic pathway. In order for it to be produced, step-by-step enzymatic reactions occur, making it crystal clear that a cluster of genes should be appropriately designed. Lactobacillus rhamnosus GG provides this glycolytic metabolic pathway, which is finally extended to the production of acetyl-CoA, a very crucial precursor of the butyrate synthesis pathway. Butyrate production can occur through two different pathways as shown below:</p>
      <div style="text-align: center;">
        <figure>
          <img src="https://static.igem.wiki/teams/4989/wiki/improved1.jpg" style="width: 600px;">
          <figcaption>Relationship between butyrate-producing bacteria and pathways for acetate and butyrate formation in Japanese patients with UC. But associated with Roseburia sp. was decreased, and that associated with Faecalibacterium sp. was maintained. Pathways for acetate and butyrate formation are from Khan et al. [31] and Duncan et al. [23], respectively.http://dx.doi.org/10.12938/bmfh.18-029</figcaption>
        </figure>
      </div>
      <p class="igemthrace-blue-text-coloring">By utilizing acetyl-CoA through the production of 7 different enzymes butyrate is produced. After extensive bibliographical research, we came up with an exact map of the gene cluster and the enzymes they produce:</p>
      <div style="text-align: center;">
        <figure>
          <img src="https://static.igem.wiki/teams/4989/wiki/circuit.png" style="width: 600px;">
          <figcaption>  </figcaption>
        </figure>
      </div>
      <p class="igemthrace-blue-text-coloring">Since Lactobacillus rhamnosus GG produces naturally acetate and after our dry lab’s bioinformatic results on which proteins are negatively enriched in people with depression, we focused on utilizing the butyryl-CoA: acetyl-CoA transferase as a final butyrate-producing pathway in which the transferase will deploy the naturally produced acetate and the butyryl-CoA produced from the gene cluster inserted into the host, for butyrate to be formed.</p>
      <p class="igemthrace-blue-text-coloring">In the iGEM’s Registry, part BBa_K1618021 was designed and entered by the 2015 iGEM team NRP-UEA-Norwich, and was pruned to produce butyrate. This part was a composite part, with the basic parts from which it was made up, designed also from the same team.</p>
      <div style="text-align: center;">
        <figure>
          <img src="https://static.igem.wiki/teams/4989/wiki/improved3.jpg" style="width: 600px;">
          <figcaption>Design of the part BBa_K1618021</figcaption>
        </figure>
      </div>
      <p class="igemthrace-blue-text-coloring">After revising the results of the characterization of this part and the sequence that was designed by this team we as well as Igem NRP-UEA-Norwich concluded that the part needed much further re-design both in the genetic and regulatory aspects in order to make it work and produce butyrate.</p>
      <p class="igemthrace-blue-text-coloring">The faults of the sequences were located in four different directions:</p>
      <ul><ul>
        <li class="igemthrace-blue-text-coloring">Some of the genes (basic parts) did not contain a start and a stop codon, something that could contribute to the overall inefficiency of the composite part.</li>
        <li class="igemthrace-blue-text-coloring">It was stated that the origin of the parts was from Coprococcus sp.L2-50 DSM. Nonetheless, there wasn’t any bibliographic reference or an NCBI quotation in order to review the sequence and ensure that each one of them encoded the correct enzyme.</li>
        <li class="igemthrace-blue-text-coloring">The sequences weren’t codon optimized for our potential hosts, Lactobacillus rhamnosus GG, Lactobacillus plantarum lp-01, Lactobacillus acidophilus DSM 20079 and E.coli K-12.</li>
        <li class="igemthrace-blue-text-coloring">The organization of the different genes in the gene clusters wasn’t cited or attributed in a specific experimental or bibliographical source, and therefore we needed to revise if the whole organization of the gene cluster was appropriate for the expression.</li>
      </ul></ul>
      <p class="igemthrace-blue-text-coloring">A separate 2016 Igem team METU_HS_Ankara had designed and entered in the Part Registry the butyryl-CoA:acetyl-CoA transferase that is our final enzyme before the production of butyrate. In the parts registry, BBa_K2052018, the sequence is well-documented and the enzyme seems to be expressed but there were some general issues in the whole part sequence:</p>
      <ul><ul>
        <li class="igemthrace-blue-text-coloring">Due to the fact that the team characterized this part, it was designed in order for it to have downstream the RFP protein and thus there was no stop codon.</li>
        <li class="igemthrace-blue-text-coloring">The sequence wasn’t codon optimized for our potential hosts, Lactobacillus rhamnosus GG, Lactobacillus plantarum lp-01, and Lactobacillus acidophilus DSM 20079  but only for E.coli.</li>
      </ul></ul>
      <h4 class="igemthrace-blue-text-coloring">The redesign process...</h4>
      <p class="igemthrace-blue-text-coloring">Going step by step through the above difficulties we managed to tackle all of them and provide a new and improved genetic makeup for our composite part.</p>
      <p class="igemthrace-blue-text-coloring">For the inclusion of starting and ending codons and the reassurance of the sequence provided, we conducted bibliographic research, ending up in a research paper which is cited in the references section [1], that provided us with the correct sequences from Coprococcus sp L2-50 DSM which were registered in the NCBI database[2].</p>
      <p class="igemthrace-blue-text-coloring">From the same research paper, we concluded that the former organization of the gene cluster was right and it is the same as the line of the genes found in Coprococcus sp L2-50 DSM. Therefore we kept it without making any changes. Codon optimization was performed by using the codon optimization tools from Genscript. For the different lactobacillus strains is not necessary to find the exact strain, as codon optimization is performed at the level of genus.</p>
      <p class="igemthrace-blue-text-coloring">For the butyryl-CoA:acetyl-CoA transferase sequence, the team provided the gene name from the NCBI database where we could recover the whole sequence, re-designing it by including a start and a stop codon. Also from Genscript’s codon optimization tool, we enhanced our sequences for the Lactobacillus strains as hosts.</p>
      <p class="igemthrace-blue-text-coloring">Generally, the former part did not contain genes that encoded a specific enzyme that converts butyryl-CoA in a specific way to butyrate. Therefore we added as a final enzymatic step the butyryl-CoA: acetyl-CoA transferase.</p>
      <p class="igemthrace-blue-text-coloring">Finally, another general concern about the sequences was the inclusion of too many restriction sites of enzymes that are commonly used for the Biobrick or the Type IIS assembly. Gensmart’s optimization tool could exclude all those restriction sites from each gene’s sequence.</p>
      <h4 class="igemthrace-blue-text-coloring">Final design of the gene cluster...</h4>
      <p class="igemthrace-blue-text-coloring">After the redesign of each gene sequence, we created our new basic parts BBa_K4989000-BBa_K4989006 and we documented them on their corresponding registry page.</p>
      <p class="igemthrace-blue-text-coloring">We had them synthesized by Twist, as gBlocks  and we included on their edges the corresponding restriction sites for the Type IIS restriction enzyme BsaI. In this way, we could perform One Pot Assembly and ligate all the gene blocks together.</p>
      <div style="text-align: center;">
        <figure>
          <img src="https://static.igem.wiki/teams/4989/wiki/improved4.jpg" style="width: 600px;">
          <figcaption>Restrictions sites for the GoldenGate assembly of our new improved part</figcaption>
        </figure>
      </div>
      <p class="igemthrace-blue-text-coloring">Concluding, the final gene cluster is formed as the picture shows bellow:</p>
      <div style="text-align: center;">
        <figure>
          <img src="https://static.igem.wiki/teams/4989/wiki/improved5.jpg" style="width: 600px;">
          <figcaption>Final gene cluster for the butyrate production</figcaption>
        </figure>
      </div>
    </div>



  
  <div class="igemthrace-space-height-2"></div>

  <div class="row igemthrace-blue-shadow p-3 rounded">
      <h2 class="igemthrace-blue-text-coloring">Engineering of the metabolic pathway-regulatory elements</h2>
      <p class="igemthrace-blue-text-coloring">Regulatory elements are highly important for the expression of the final product of each gene. The former sequence of the part BBa_K1618021 included a LacI promoter and no other cis-regulatory elements. In this case, it is obvious that the expression would be inducible (by lactose) and the expression could be seriously jeopardized due to the lack of a specific sequence that would guarantee the production of the target enzymes and thus butyrate. So the issues we came up with were:</p>
      <ul><ul>
        <li class="igemthrace-blue-text-coloring">The presence of an inducible lactose promoter</li>
        <li class="igemthrace-blue-text-coloring">The absence of Ribosome Binding Sites (RBS) upstream of every gene</li>
        <li class="igemthrace-blue-text-coloring">General regulatory systems that are designed specifically for enhancing the expression of coding sequences in Lactobacillus strains.</li>
      </ul></ul>
      <h4 class="igemthrace-blue-text-coloring">The redesign process...</h4>
      <p class="igemthrace-blue-text-coloring">First of all, the regulatory systems must be compatible with our target microorganism host, Lactobacillus rhamnosus GG. The designed metabolic pathway has to be expressed constitutively, and therefore we searched for a constitutive promoter that is compatible and recognizable from the Lactobacillus strain. We ended up utilizing the BBa_K2253001 part that represents the P32 constitutive promoter and an RBS site, specifically designed for the Lactobacillus genus. </p>
      <p class="igemthrace-blue-text-coloring">The addition of a terminator is optional since the insert of the improved part in an expression vector might end up having downstream a terminator. In case this isn’t the scenario we propose the BBa_K2978211 part which represents a terminator specifically designed for the Lactobacillus strains.</p>
      <p class="igemthrace-blue-text-coloring">For the protein expression, we added RBSs in front of every gene except for the first one that already has a ribosome binding site from the promoter P32 part. The BBa_B0034 part represents the RBS sequence that we used in between the parts.</p>
      <p class="igemthrace-blue-text-coloring">All the above parts are well-characterized and that was one of the criteria that led us to use those parts. As mentioned above, each gene is synthesized from Twist as gene blocks. Due to the length limit, we synthesized each gene together with the RBS (the first one also included the promoter). So, we added the appropriate scars that make the part compatible with the Biobrick standards.</p>
  </div>



  
  
  <div class="igemthrace-space-height-2"></div>

  <div class="row igemthrace-blue-shadow p-3 rounded">
      <h2 class="igemthrace-blue-text-coloring">Plasmid design - Food grade system</h2>
      <p class="igemthrace-blue-text-coloring">In order for the project to be safe we need to construct an expression vector, that has the following characteristics:</p>
      <ul><ul>
        <li class="igemthrace-blue-text-coloring">It is composed only of DNA that originates from the same organism (self-cloning) or from an organism that is Generally Recognized As Safe (GRAS) and is closely related to the target host.</li>
        <li class="igemthrace-blue-text-coloring">It does not contain an antibiotic resistance marker but a food-grade marker that can be either dominant or complimentary.</li>
        <li class="igemthrace-blue-text-coloring">Contains regulatory components that are compatible with the expression in the Lactobacillus strain.</li>
      </ul></ul>
      <p class="igemthrace-blue-text-coloring">After thorough bibliographic research, we ended up re-designing the plasmid pMG36e. Unfortunately, this vector is not easily available for ordering and the sequence provided for order wasn’t quite the ideal one since it included an antibiotic resistance marker that wasn’t easily removed as well as the restriction enzymes weren’t compatible with any assembly method. So we found the plasmid’s sequence from a trusting firm that had the plasmid for order and redesigned it as shown below:</p>
      <ul><ul>
        <li class="igemthrace-blue-text-coloring">We excluded the antibiotic resistance gene and replaced it with the nisI gene. The nisI gene is a dominant and safe marker that expresses a peptide that grants immunity to the bacteria that are transformed with the plasmid.</li>
        <li class="igemthrace-blue-text-coloring">We excluded a small coding sequence (ORF) that was after the P32 promoter.</li>
        <li class="igemthrace-blue-text-coloring">We had our insert designed just as was mentioned before with the difference of excluding the promoter and including the terminator that was specifically designed for Lactobacillus strains.</li>
        <li class="igemthrace-blue-text-coloring">Since we aimed to synthesize the whole sequence we would not perform digestions. Therefore we included the Biobrick scars between the genes and the RBS as well as between the insert and the plasmid.</li>
        <li class="igemthrace-blue-text-coloring">Also, since the selection of the plasmids cannot be done directly through the nisin selection marker, we included by the end of the butyryl-CoA:acetyl-CoA transferase the gene of the GFP protein. In this way, we can discriminate the colonies that have the recombinant plasmid from those that do not have it.</li>
      </ul></ul>
      <div style="text-align: center;">
        <figure>
          <img src="https://static.igem.wiki/teams/4989/wiki/improved6.png" style="width: 600px;">
          <figcaption>  </figcaption>
        </figure>
      </div>
      <p class="igemthrace-blue-text-coloring"></p>
      <p class="igemthrace-blue-text-coloring"></p>
      <p class="igemthrace-blue-text-coloring"></p>
      <p class="igemthrace-blue-text-coloring"></p>
      <p class="igemthrace-blue-text-coloring"></p>
      <p class="igemthrace-blue-text-coloring"></p>
      <p class="igemthrace-blue-text-coloring"></p>
      <p class="igemthrace-blue-text-coloring"></p>
  </div>


  
  
  
  <div class="igemthrace-space-height-2"></div>

  <div class="row igemthrace-blue-shadow p-3 rounded">
      <h2 class="igemthrace-blue-text-coloring">Alternative plasmid design - Food grade system</h2>
      <p class="igemthrace-blue-text-coloring">Although the pMG36e plasmid seems to be the ideal vector for the cloning of the butyrate-producing gene cluster, some technical issues with the production of the vector arose that led us to search for a different option. In order to properly inform the future Igem teams that intend to synthesize this vector we indicate below the technical issues that we had to face:</p>
      <ul><ul>
        <li class="igemthrace-blue-text-coloring">Due to the extended size of the plasmid, the financial cost of its synthesis was extremely high as well as the time required for it to be created.</li>
        <li class="igemthrace-blue-text-coloring">The sequence of the plasmid, in some points showed high complexity, in order for the DNA synthesis company to proceed with its production. Therefore, this difficulty required a change to critical parts of the plasmid.</li>
        <li class="igemthrace-blue-text-coloring">We could not ask for the synthesis of a plasmid that did not contain an antibiotic-resistance gene. This was something that made its synthesis even harder due to the fact that even if we included such sequence and removed it later with proper digestion, the size of the plasmid would get larger and the turnaround time as well as the cost for the synthesis would arise</li>
      </ul></ul>
      <p class="igemthrace-blue-text-coloring">So, since we encountered such difficulties we decided to use a different kind of plasmid that is also food-graded. The plasmid is part of the pNZ series, which includes a wide variety of plasmids used specifically for food-grade systems expressed in the Lactobacillus and other related genera.</p>
      <div style="text-align: center;">
        <figure>
          <img src="https://static.igem.wiki/teams/4989/wiki/improved7.png" style="width: 600px;">
          <figcaption>  </figcaption>
        </figure>
      </div>
      <p class="igemthrace-blue-text-coloring">This plasmid is not ideal for our cloning strategy but fits just as well for the insert’s expression. Also, it was available from our lab and in very good condition. Below, we can see the difficulties posed and the solution we implemented by the use of this vector:</p>
      <ul><ul>
        <li class="igemthrace-blue-text-coloring">The vector includes an inducible promoter from nisin PnisA. Since we don’t want our coding sequence to be expressed under an inducible but a constitutive promoter, we included in our part’s insert sequence the P32 promoter and didn’t use nisin. In this way, the PnisA promoter is not active.</li>
        <li class="igemthrace-blue-text-coloring">The lacF marker is complimentary and therefore it would not be possible to distinguish between the colonies that received the recombinant plasmid and those that received just the plasmid. Therefore, we maintained the GFP gene by the end of the transferase’s gene sequence.</li>
        <li class="igemthrace-blue-text-coloring">The plasmid already includes a terminator. So, we re-designed our new improved part without a terminator.</li>
      </ul></ul>
      <p class="igemthrace-blue-text-coloring"></p>
      <p class="igemthrace-blue-text-coloring"></p>
      <p class="igemthrace-blue-text-coloring"></p>
      <p class="igemthrace-blue-text-coloring"></p>
      <p class="igemthrace-blue-text-coloring"></p>
      <p class="igemthrace-blue-text-coloring"></p>
      <p class="igemthrace-blue-text-coloring"></p>
      <p class="igemthrace-blue-text-coloring"></p>
  </div>


    
  
  <div class="igemthrace-space-height-2"></div>

  <div class="row igemthrace-blue-shadow p-3 rounded">
      <h2 class="igemthrace-blue-text-coloring">Conclusions</h2>
      <p class="igemthrace-blue-text-coloring">In a short summary, our improvements of the old part BBa_K1618021 are:</p>
      <ul><ul>
        <li class="igemthrace-blue-text-coloring">The change of gene sequence (whole change, addition and of start and stop codons, codon optimization).</li>
        <li class="igemthrace-blue-text-coloring">Addition of the butyryl-CoA:acetyl-CoA transferase at the end of the gene cluster.</li>
        <li class="igemthrace-blue-text-coloring">Addition of RBS upstream of each gene.</li>
        <li class="igemthrace-blue-text-coloring">Replacement of the Plac promoter with the P32 promoter.</li>
        <li class="igemthrace-blue-text-coloring">Addition of a terminator designed for Lactobacillus.</li>
      </ul></ul>
  </div>


      
  
  <div class="igemthrace-space-height-2"></div>

  <div class="row igemthrace-blue-shadow p-3 rounded">
      <h2 class="igemthrace-blue-text-coloring">General Concerns</h2>
      <p class="igemthrace-blue-text-coloring">Our designed part seems very promising in expressing our target molecule butyrate. Yet there are some concerns that we should reconsider in order for the project to be more able to work:</p>
      <ul><ul>
        <li class="igemthrace-blue-text-coloring">There should be more profound work on finding out if specific 5’ UTR sequences (Shine-Dalgarno), are needed between the P32 promoter and the first gene of our path.</li>
        <li class="igemthrace-blue-text-coloring">The pMG36e plasmid which is widely used and has all the characteristics that we want, also exhibits a large size that reaches an almost prohibitive limit for the transfection of competent cells. The same applies to the pNZ8149 plasmid which is in a better size and has more chances for transformation. Thus, we need to revise if the insertion of all the gene clusters is the only and ideal solution or there is an alternative on separating the cluster in two plasmids or performing the transfection with another method.</li>
        <li class="igemthrace-blue-text-coloring">The parts sequences were extensive in regards with the size and sometimes their complexity. This can be prohibitive for some companies to synthesize it since it may not pass the complexity test. Therefore, more research and redesign should be conducted in order to achieve less complexity issues.</li>
      </ul></ul>
      <p class="igemthrace-blue-text-coloring"></p>
      <p class="igemthrace-blue-text-coloring"></p>
      <p class="igemthrace-blue-text-coloring"></p>
      <p class="igemthrace-blue-text-coloring"></p>
      <p class="igemthrace-blue-text-coloring"></p>
      <p class="igemthrace-blue-text-coloring"></p>
      <p class="igemthrace-blue-text-coloring"></p>
      <p class="igemthrace-blue-text-coloring"></p>
  </div>

    
  <div class="igemthrace-space-height-2"></div>

  <div class="row igemthrace-blue-shadow p-3 rounded">
      <h2 class="igemthrace-blue-text-coloring">References</h2>
      <ul><ul>
        <li class="igemthrace-blue-text-coloring">Petra Louis, Sheila I. McCrae, Cédric Charrier, Harry J. Flint, Organization of butyrate synthetic genes in human colonic bacteria: phylogenetic conservation and horizontal gene transfer, FEMS Microbiology Letters, Volume 269, Issue 2, April 2007, Pages 240–247,https://doi.org/10.1111/j.1574-6968.2006.00629.x </li>
        <li class="igemthrace-blue-text-coloring">Butyrate-Producing Bacterium L2-50 Putative Fe-S Oxidoreductase Gene, Partial Cds; Thiolase, Crotonase, Beta Hydroxybutyryl-CoA Dehydrogenase, Butyryl-CoA Dehydrogenase, Electron Transfer Flavoprotein Beta-Subunit, and Electron Transfer Flavoprotein Alpha-Subunit Genes, Complete Cds; and Putative Multidrug Efflux Pump Gene, Partial Cds.” NCBI Nucleotide, July 2016, https://www.ncbi.nlm.nih.gov/nuccore/DQ987697.1/</li>
        <li class="igemthrace-blue-text-coloring">Landete, José Maria. “A Review of Food-Grade Vectors in Lactic Acid Bacteria: From the Laboratory to Their Application.” Critical Reviews in Biotechnology, vol. 37, no. 3, Feb. 2016, pp. 296–308, https://doi.org/10.3109/07388551.2016.1144044. Accessed 17 Mar. 2022.</li>
        <li class="igemthrace-blue-text-coloring">A Review of Food-Grade Vectors in Lactic Acid Bacteria: From the Laboratory to Their Application.” Critical Reviews in Biotechnology, vol. 37, no. 3, Feb. 2016, pp. 296–308, https://doi.org/10.3109/07388551.2016.1144044.</li>
        <li class="igemthrace-blue-text-coloring">T., Takala, and Saris P. “A Food-Grade Cloning Vector for Lactic Acid Bacteria Based on the Nisin Immunity Gene NisI.” Applied Microbiology and Biotechnology, vol. 59, no. 4-5, Jan. 2002, pp. 467–71, https://doi.org/10.1007/s00253-002-1034-4. Accessed 3 Apr. 2021.</li>
        <li class="igemthrace-blue-text-coloring">Tagliavia, Marcello, and Aldo Nicosia. “Advanced Strategies for Food-Grade Protein Production: A New E. Coli/Lactic Acid Bacteria Shuttle Vector for Improved Cloning and Food-Grade Expression.” Microorganisms, vol. 7, no. 5, Apr. 2019, p. 116, https://doi.org/10.3390/microorganisms7050116.</li>
      </ul></ul>
  </div>


</div>

<!--
<div class="row mt-4">
  <div class="col">
    <div class="bd-callout bd-callout-info">
      <h4>Silver Medal Criterion #1</h4>
      <p>Demonstrate engineering success in a part of your project by going through at least one iteration of the engineering design cycle. This achievement should be distinct from your Contribution for Bronze.<p>
      <p>If you plan to show engineering success by creating a new Part that has been shown to work as expected, you must document your contribution on the Part's Main Page on the <a href="http://parts.igem.org/Main_Page">Registry</a> for your team to be eligible for this criteria.</p>
      <hr>
      <p>Please see the <a href="https://competition.igem.org/judging/medals">2023 Medals Page</a> for more information.</p>
    </div>
  </div>
</div>
-->
{% endblock %}
